_build/detail.hbs

```diff
@@ -26,15 +26,20 @@
       <h4>Additional info</h4>
       <p>{{{mdExternal AdditionalInfoExternal}}}</p>
       {{/if}}    
- 
-      {{#if EOBrowser}}
-      <h4>Explore</h4>
-      <p><a href="{{EOBrowser}}" target="_blank"><button class="label label-info" > Open in EO Browser </button></a> </p>
-      {{/if}}  
+
       {{#if EDCBrowser}}
       <h4>Explore</h4>
-      <p><a href="{{EDCBrowser}}" target="_blank"><button class="label label-info" > Open in EDC Browser </button></a> </p>
-      {{/if}} 
+      <p><a href="{{EDCBrowser}}" target="_blank"><button class="label label-info"> Open in EDC Browser </button></a>
+            {{#if EOBrowser}}
+            <a href="{{EOBrowser}}" target="_blank"><button class="label label-info" style="margin:10px"> Open in EO Browser </button></a> </p>
```

Comment:
Sorry for noticing it only now.

The `</p>` should be out of the `if` part, so the `<p>` gets closed even if there is no EOBrowser link.

I formatted the code a bit differently so it's more visible:
```handlebars
      {{#if EDCBrowser}}
      <h4>Explore</h4>
      <p>
            <a href="{{EDCBrowser}}" target="_blank"><button class="label label-info"> Open in EDC Browser </button></a>
            {{#if EOBrowser}}
            <a href="{{EOBrowser}}" target="_blank"><button class="label label-info" style="margin:10px"> Open in EO Browser </button></a>
            {{/if}}
      </p>
      {{else}}
            {{#if EOBrowser}}
            <h4>Explore</h4>
            <p><a href="{{EOBrowser}}" target="_blank"><button class="label label-info" > Open in EO Browser </button></a> </p>
            {{/if}}
      {{/if}}
```

Reply:
Thanks a lot. This is my first time dealing with handlebar. Sorry for the poor formatting. I update the formatting.

Reply:
No problem, we all learn as we go. It looks great now.